src/cli.js

```diff
@@ -28,9 +30,17 @@ if (program.init) {
   inquirer.prompt()
     .then((answers) => {
       console.log('Saving options into the configuration file');
-      console.log(JSON.stringify(answers, null, 2));
       Object.keys(answers).forEach(k => configStore.set(k, answers[k]));
+      answers.token = 'hidden';
+      console.log(JSON.stringify(answers, null, 2));
     });
 } else {
-  console.log('Add support for calling github.js here');
+  const config = require(`${os.homedir()}/.config/configstore/standup-helper.json`);
```

Comment:
You need to use `configStore.get()` to get the properties. If you read them from a file you are bypassing the library we use for that.